st2actions/st2actions/runners/httprunner.py

```diff
@@ -204,3 +223,16 @@ def _normalize_headers(self, headers):
             result[key.lower()] = value
 
         return result
+
+    def _is_json_content(self):
+        normalized = self._normalize_headers(self.headers)
+        return normalized.get('content-type', None) == 'application/json'
+
+    def _cast_object(self, value):
```

Comment:
Hm, IIRC, we do similar thing in multiple places. Maybe it would be a good idea to make this a utility function.